.gitignore

```diff
@@ -53,3 +53,4 @@ google-services.json
 freeline.py
 freeline/
 freeline_project_description.json
+!/.idea/
```

Comment:
whole `.idea/` is not excluded because it contents the project code styles

Reply:
Ohk get it,
What about the "caches", it should be in the .gitignore?

Reply:
Already Update that @See develop branch https://github.com/UdacityAndroidDevScholarship/blood-donation/blob/f193c9f5eb733b41358565c3d0d50a142d95835a/.gitignore#L42